livekit-agents/livekit/agents/job.py

```diff
@@ -183,7 +190,8 @@ def add_participant_entrypoint(
             [JobContext, rtc.RemoteParticipant], Coroutine[None, None, None]
         ],
         *_,
```

Comment:
what is the diff between *_ and *?

Reply:
IDE complains about *, maybe when it's not the first arg?